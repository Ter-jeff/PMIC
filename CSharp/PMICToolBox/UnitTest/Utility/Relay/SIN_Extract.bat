@echo off
start C:\Jeffli\SVN\trunk\AutoGen_Team\Automation\UnitTest\WinMerge\WinMergeU.exe C:\Jeffli\SVN\trunk\AutoGen_Team\Automation\UnitTest\Utility\Relay\Expected\SIN_Extract.txt C:\Jeffli\SVN\trunk\AutoGen_Team\Automation\UnitTest\Utility\Relay\Output\SIN_Extract.txt
