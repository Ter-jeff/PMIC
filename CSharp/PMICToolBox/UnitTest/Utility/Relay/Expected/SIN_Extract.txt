Public Function SIN_Extract(Trace As Double) As String
Select Case Trace
'___U7102
	Case VOUT0A_UVI80_S_To_VOUT0_A_H10:SIN_Extract ="VOUT0A_UVI80_S_S1"
	Case VOUT0A_UVI80_S_To_VOUT1_A_J10:SIN_Extract ="VOUT0A_UVI80_S_S2"
	Case VOUT0A_UVI80_S_To_VOUT2_A_G10:SIN_Extract ="VOUT0A_UVI80_S_S3"
	Case VOUT0A_UVI80_S_To_VOUT3_A_K10:SIN_Extract ="VOUT0A_UVI80_S_S4"
	Case VOUT4A_UVI80_S_To_VOUT4_A_F10:SIN_Extract ="VOUT4A_UVI80_S_S5"
	Case VOUT4A_UVI80_S_To_VOUT5_A_F8:SIN_Extract ="VOUT4A_UVI80_S_S6"
	Case VOUT4A_UVI80_S_To_VOUT6_A_E10:SIN_Extract ="VOUT4A_UVI80_S_S7"
	Case VOUT4A_UVI80_S_To_VOUT7_A_E9:SIN_Extract ="VOUT4A_UVI80_S_S8"
'___U7101
	Case VOUT0B_UVI80_S_To_VOUT0_B_H13:SIN_Extract ="VOUT0B_UVI80_S_S1"
	Case VOUT0B_UVI80_S_To_VOUT1_B_J13:SIN_Extract ="VOUT0B_UVI80_S_S2"
	Case VOUT0B_UVI80_S_To_VOUT2_B_G13:SIN_Extract ="VOUT0B_UVI80_S_S3"
	Case VOUT0B_UVI80_S_To_VOUT3_B_K13:SIN_Extract ="VOUT0B_UVI80_S_S4"
	Case VOUT4B_UVI80_S_To_VOUT4_B_F13:SIN_Extract ="VOUT4B_UVI80_S_S5"
	Case VOUT4B_UVI80_S_To_VOUT5_B_F7:SIN_Extract ="VOUT4B_UVI80_S_S6"
	Case VOUT4B_UVI80_S_To_VOUT6_B_E13:SIN_Extract ="VOUT4B_UVI80_S_S7"
	Case VOUT4B_UVI80_S_To_VOUT7_B_E6:SIN_Extract ="VOUT4B_UVI80_S_S8"
'___U7104
	Case VOUT8A_UVI80_S_To_VOUT8_A_D10:SIN_Extract ="VOUT8A_UVI80_S_S1"
	Case VOUT8A_UVI80_S_To_VOUT9_A_D9:SIN_Extract ="VOUT8A_UVI80_S_S2"
	Case VOUT8A_UVI80_S_To_VOUT10_A_C10:SIN_Extract ="VOUT8A_UVI80_S_S3"
	Case VOUT8A_UVI80_S_To_VOUT11_A_C9:SIN_Extract ="VOUT8A_UVI80_S_S4"
	Case VOUT12A_UVI80_S_To_VOUT12_A_B10:SIN_Extract ="VOUT12A_UVI80_S_S5"
	Case VOUT12A_UVI80_S_To_VOUT13_A_F11:SIN_Extract ="VOUT12A_UVI80_S_S6"
	Case VOUT12A_UVI80_S_To_VOUT14_A_D2:SIN_Extract ="VOUT12A_UVI80_S_S7"
	Case VOUT12A_UVI80_S_To_VOUT15_A_C2:SIN_Extract ="VOUT12A_UVI80_S_S8"
'___U7103
	Case VOUT8B_UVI80_S_To_VOUT8_B_D13:SIN_Extract ="VOUT8B_UVI80_S_S1"
	Case VOUT8B_UVI80_S_To_VOUT9_B_D6:SIN_Extract ="VOUT8B_UVI80_S_S2"
	Case VOUT8B_UVI80_S_To_VOUT10_B_C13:SIN_Extract ="VOUT8B_UVI80_S_S3"
	Case VOUT8B_UVI80_S_To_VOUT11_B_C6:SIN_Extract ="VOUT8B_UVI80_S_S4"
	Case VOUT12B_UVI80_S_To_VOUT12_B_B13:SIN_Extract ="VOUT12B_UVI80_S_S5"
	Case VOUT12B_UVI80_S_To_VOUT13_B_F12:SIN_Extract ="VOUT12B_UVI80_S_S6"
	Case VOUT12B_UVI80_S_To_VOUT14_B_D5:SIN_Extract ="VOUT12B_UVI80_S_S7"
	Case VOUT12B_UVI80_S_To_VOUT15_B_C5:SIN_Extract ="VOUT12B_UVI80_S_S8"
'___U7106
	Case VOUT16A_UVI80_S_To_VOUT16_A_E2:SIN_Extract ="VOUT16A_UVI80_S_S1"
	Case VOUT16A_UVI80_S_To_VOUT17_A_B2:SIN_Extract ="VOUT16A_UVI80_S_S2"
	Case VOUT16A_UVI80_S_To_VOUT18_A_G8:SIN_Extract ="VOUT16A_UVI80_S_S3"
	Case VOUT16A_UVI80_S_To_VOUT19_A_G9:SIN_Extract ="VOUT16A_UVI80_S_S4"
	Case VOUT20A_UVI80_S_To_VOUT20_A_H9:SIN_Extract ="VOUT20A_UVI80_S_S5"
	Case VOUT20A_UVI80_S_To_VOUT21_A_J9:SIN_Extract ="VOUT20A_UVI80_S_S6"
	Case VOUT20A_UVI80_S_To_VOUT22_A_F2:SIN_Extract ="VOUT20A_UVI80_S_S7"
	Case VOUT20A_UVI80_S_To_VOUT23_A_G2:SIN_Extract ="VOUT20A_UVI80_S_S8"
'___U7105
	Case VOUT16B_UVI80_S_To_VOUT16_B_E5:SIN_Extract ="VOUT16B_UVI80_S_S1"
	Case VOUT16B_UVI80_S_To_VOUT17_B_B5:SIN_Extract ="VOUT16B_UVI80_S_S2"
	Case VOUT16B_UVI80_S_To_VOUT18_B_G7:SIN_Extract ="VOUT16B_UVI80_S_S3"
	Case VOUT16B_UVI80_S_To_VOUT19_B_G6:SIN_Extract ="VOUT16B_UVI80_S_S4"
	Case VOUT20B_UVI80_S_To_VOUT20_B_H6:SIN_Extract ="VOUT20B_UVI80_S_S5"
	Case VOUT20B_UVI80_S_To_VOUT21_B_J6:SIN_Extract ="VOUT20B_UVI80_S_S6"
	Case VOUT20B_UVI80_S_To_VOUT22_B_F5:SIN_Extract ="VOUT20B_UVI80_S_S7"
	Case VOUT20B_UVI80_S_To_VOUT23_B_G5:SIN_Extract ="VOUT20B_UVI80_S_S8"
'___U7107
	Case VOUT24A_UVI80_S_To_VOUT24_A_H2:SIN_Extract ="VOUT24A_UVI80_S_S1"
	Case VOUT24A_UVI80_S_To_VOUT25_A_J2:SIN_Extract ="VOUT24A_UVI80_S_S2"
	Case VOUT24A_UVI80_S_To_VOUT26_A_K2:SIN_Extract ="VOUT24A_UVI80_S_S3"
	Case VOUT24A_UVI80_S_To_VOUT27_A_F3:SIN_Extract ="VOUT24A_UVI80_S_S4"
	Case VOUT24B_UVI80_S_To_VOUT24_B_H5:SIN_Extract ="VOUT24B_UVI80_S_S5"
	Case VOUT24B_UVI80_S_To_VOUT25_B_J5:SIN_Extract ="VOUT24B_UVI80_S_S6"
	Case VOUT24B_UVI80_S_To_VOUT26_B_K5:SIN_Extract ="VOUT24B_UVI80_S_S7"
	Case VOUT24B_UVI80_S_To_VOUT27_B_F4:SIN_Extract ="VOUT24B_UVI80_S_S8"
End Select
Exit Function
End Function
